keyboards/handwired/prime_exl_plus/keymaps/default/keymap.c

```diff
@@ -0,0 +1,93 @@
+		KC_TAB, 	KC_Q, 		KC_W, 		KC_E, 		KC_R, 		KC_T,				  KC_GRV, 	KC_Q, 		KC_W, 		KC_E, 		KC_R, 		KC_T,			KC_Y, 		KC_U, 		KC_I, 		KC_O, 		KC_P, 	  KC_LBRC,    KC_RBRC,
+		KC_TAB,	  KC_A,		  KC_S,		  KC_D,	  	KC_F,		  KC_G,			    KC_TAB, 	KC_A, 		KC_S, 		KC_D, 		KC_F, 		KC_G,			KC_H, 		KC_J, 		KC_K, 		KC_L, 		KC_QUOT, 	            KC_ENT,
+		KC_LSFT,	KC_Z,		  KC_X,		  KC_C,		  KC_V,		  KC_V,			    KC_LSFT, 	KC_Z, 		KC_X, 		KC_C, 		KC_V, 		KC_B,			MO(3), 		KC_N, 		KC_M, 		KC_COMM,	KC_DOT, 	KC_SLSH,   	KC_LSFT,
+		KC_LCTL,	KC_LALT,	TO(0),		KC_P,	    KC_H,		  KC_SPC,       KC_LCTL, 	KC_LGUI, 	                    KC_LALT, 	MO(2),	  KC_SPC, 	KC_MINS,  			                        KC_RGUI,  	KC_RCTL
+    ),
+
+	LAYOUT(
+		KC_NLCK, 	          KC_TRNS, 	KC_TRNS, 	KC_TRNS, 	KC_TRNS, 	KC_TRNS, 				KC_TRNS, 		KC_TRNS, 		KC_TRNS, 		KC_TRNS, 		KC_TRNS, 		KC_TRNS,    KC_TRNS, 	  KC_TRNS, 	  KC_TRNS, 		KC_TRNS, 	  KC_TRNS, 	  KC_DEL,
+		RGB_TOG, 	          RGB_MOD, 	KC_TRNS, 	KC_TRNS, 	KC_TRNS, 	KC_TRNS,				KC_TRNS, 		KC_TRNS, 		KC_TRNS, 		KC_TRNS, 		KC_TRNS, 		KC_TRNS,		KC_TRNS,  	KC_TRNS, 	  KC_UP, 	    KC_TRNS, 	  KC_SCLN, 	  KC_BSLS,    KC_TRNS,
+		RGB_VAI,	          RGB_VAD,	KC_TRNS,	KC_TRNS,	KC_TRNS,	KC_TRNS,				KC_CAPS, 		KC_TRNS, 		KC_TRNS, 		KC_TRNS, 		KC_TRNS, 		KC_TRNS,		KC_TRNS,  	KC_LEFT, 	  KC_DOWN, 		KC_RGHT,  	KC_TRNS, 	              KC_TRNS,
+		RGB_MODE_RAINBOW,	  KC_TRNS,	KC_TRNS,	KC_TRNS,	KC_TRNS,	KC_TRNS,				KC_TRNS, 		KC_TRNS, 		KC_TRNS, 		KC_TRNS, 	  KC_TRNS,    KC_TRNS,		KC_TRNS, 		KC_TRNS, 		KC_TRNS,  	KC_TRNS, 	  KC_TRNS,    KC_TRNS,    KC_TRNS,
+		KC_TRNS,            KC_TRNS,	KC_TRNS,	KC_TRNS,	KC_TRNS,	KC_TRNS,        KC_TRNS,    KC_TRNS,                            KC_TRNS,    KC_TRNS,    KC_TRNS,    KC_TRNS,                                        KC_TRNS,    KC_TRNS
+    ),
+
+	LAYOUT(
+		KC_TRNS, 	KC_TRNS, 	KC_TRNS, 	KC_TRNS, 	KC_TRNS, 	KC_TRNS, 				KC_TRNS, 		KC_TRNS, 		KC_TRNS, 		KC_TRNS, 		KC_TRNS, 		KC_TRNS,				 		KC_TRNS, 	KC_TRNS, 	KC_TRNS, 	KC_TRNS, 	KC_TRNS, 	KC_TRNS,
+		KC_TRNS, 	KC_TRNS, 	KC_TRNS, 	KC_TRNS, 	KC_TRNS, 	KC_TRNS,				KC_TRNS, 		KC_TRNS, 		KC_TRNS, 		KC_TRNS, 		KC_TRNS, 		KC_TRNS,			 	 		KC_TRNS, 	KC_TRNS, 	KC_TRNS, 	KC_TRNS, 	KC_TRNS, 	KC_TRNS,  KC_TRNS,
+		KC_TRNS,	KC_TRNS,	KC_TRNS,	KC_TRNS,	KC_TRNS,	KC_TRNS,				KC_TRNS, 		KC_TRNS, 		KC_TRNS, 		KC_TRNS, 		KC_TRNS, 		KC_TRNS,				 		KC_TRNS, 	KC_TRNS, 	KC_TRNS, 	KC_TRNS,	KC_TRNS,           	KC_TRNS,
+		KC_TRNS,	KC_TRNS,	KC_TRNS,	KC_TRNS,	KC_TRNS,	KC_TRNS,				KC_TRNS, 		KC_TRNS, 		KC_TRNS, 		KC_TRNS, 	  KC_TRNS,		KC_TRNS,						KC_TRNS,  KC_TRNS, 	KC_TRNS, 	KC_TRNS, 	KC_TRNS,  KC_TRNS,  KC_TRNS,
+		KC_TRNS,	KC_TRNS,	KC_TRNS,	KC_TRNS,	KC_TRNS,	KC_TRNS,        KC_TRNS,    KC_TRNS,                            KC_TRNS,    KC_TRNS,            KC_TRNS,  KC_TRNS,                                KC_TRNS,  KC_TRNS
+    )
+};
```

Comment:
For Lines 18-50:

```c
const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
    LAYOUT(
        KC_ESC,  KC_LPRN, KC_RPRN, KC_PSLS, KC_PAST, KC_BSPC,     KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_BSPC,
        KC_F1,   KC_F2,   KC_P7,   KC_P8,   KC_P9,   KC_DEL,      KC_GRV,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC,
        KC_F3,   KC_F4,   KC_P4,   KC_P5,   KC_P6,   KC_PMNS,     KC_TAB,  KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_QUOT,          KC_ENT,
        KC_F5,   KC_F6,   KC_P1,   KC_P2,   KC_P3,   KC_PPLS,     KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    MO(3),   KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT,
        KC_F7,   KC_F8,   TO(1),   KC_PDOT, KC_P0,   KC_PENT,     KC_LCTL, KC_LGUI,          KC_LALT,          MO(2),   KC_SPC,           KC_MINS,                   KC_RGUI, KC_RCTL
    ),

    LAYOUT(
        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_F9,       KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_BSPC,
        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,        KC_GRV,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC,
        KC_TAB,  KC_A,    KC_S,    KC_D,    KC_F,    KC_G,        KC_TAB,  KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_QUOT,          KC_ENT,
        KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_V,        KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    MO(3),   KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT,
        KC_LCTL, KC_LALT, TO(0),   KC_P,    KC_H,    KC_SPC,      KC_LCTL, KC_LGUI,          KC_LALT,          MO(2),   KC_SPC,           KC_MINS,                   KC_RGUI, KC_RCTL
    ),

    LAYOUT(
        KC_NLCK, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,     KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_DEL,
        RGB_TOG, RGB_MOD, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,     KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_UP,   KC_TRNS, KC_SCLN, KC_BSLS, KC_TRNS,
        RGB_VAI, RGB_VAD, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,     KC_CAPS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_LEFT, KC_DOWN, KC_RGHT, KC_TRNS,          KC_TRNS,
        RGB_M_R, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,     KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,     KC_TRNS, KC_TRNS,          KC_TRNS,          KC_TRNS, KC_TRNS,          KC_TRNS,                   KC_TRNS, KC_TRNS
    ),

    LAYOUT(
        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,     KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,     KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,     KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,
        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,     KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,     KC_TRNS, KC_TRNS,          KC_TRNS,          KC_TRNS, KC_TRNS,          KC_TRNS,                   KC_TRNS, KC_TRNS
    )
};
```

Four-space indent and change `RGB_MODE_RAINBOW` to `RGB_M_R`.

No logic changes.